test/runtests.sh

```diff
@@ -12,8 +12,19 @@ fi
 # compile the code into the bin folder
 javac  ../src/seedu/addressbook/AddressBook.java -d ../bin
 
+# run the program and pass in invalid file paths
```

Comment:
I think @damithc wanted something like:
```
# (invalid) no parent directory, invalid filename with no extension
# (invalid) invalid parent directory that does not exist, valid filename
# (invalid) no parent directory, invalid filename with dot on 1st character
```
etc.

You can also arrange the tests according to them (e.g. put the `no parent directory` tests together)